doc/INSTALL-WINDOWS

```diff
@@ -0,0 +1,57 @@
+Please, read INSTALL for general information about installing John on your
+system.
+
+== Preamble
+
+    Whenever you feel like giving up, think of all the people that would love
+    to see you fail.
+
+== How to install on Windows
+
+    The steps listed below might work unchanged (untested).
+
+=== Preconditions and Required stuff
+
+    - Install CygWin (see https://cygwin.com/install.html)
+
+    - Open the Command Prompt in Windows (see [1])
+
+    - Install required CygWin packages.
+      - C:\cygwin64\setup-x86_64.exe -q -P libssl-devel -P libbz2-devel
+      - C:\cygwin64\setup-x86_64.exe -q -P libgmp-devel -P zlib-devel"
+      - C:\cygwin64\setup-x86_64.exe -q -P libOpenCL-devel -P libcrypt-devel
+
+    - Open a CygWin terminal. To do that, inside the Command Prompt, type:
+      - C:\cygwin64\bin\bash
+
+=== Get latest bleeding-edge Jumbo and build
+
+==== Clone GIT repo
+
+    Download https://github.com/magnumripper/JohnTheRipper/archive/bleeding-jumbo.zip
```

Comment:
That's not "cloning a git repo", and most windows users wouldn't want to clone the git repo, so I think the heading should be rephrased.

Reply:
Good catch. Not super important but this might lead to confusion. What would be a better heading, `==== Download latest snapshot of source` perhaps?

Reply:
I'd just say "get" instead of "download", since it describes downloading and unzipping.